README.md

```diff
@@ -2,13 +2,24 @@
 
 This is _Datawrapper Core_, the open source platform that powers the charting tools hosted as free service on [datawrapper.de](https://datawrapper.de).
 
-Most of the functionality in Datawrapper is provided by plugins, [some of which are open source](https://github.com/datawrapper?utf8=%E2%9C%93&q=plugin-&type=public&language=) but the majority is not. However, you are free to [create your own visualization plugins](https://github.com/datawrapper/datawrapper/wiki/Extending-Datawrapper).
+Most of the functionality in Datawrapper is provided by plugins, [some of which are open source](https://github.com/datawrapper?utf8=%E2%9C%93&q=plugin-&type=public&language=) but the majority is not. 
 
-Datawrapper was created by [Mirko Lorenz](http://www.mirkolorenz.com/), [Nicolas Kayser-Bril](http://nkb.fr) and [Gregor Aisch](http://driven-by-data.net/) and has in the past received funding by [ABZV](http://www.abzv.de/).
+Note that significant parts of the Datawrapper core have been rewritten in Node.js already, you can find them in the following repositories:
+
+- [api](https://github.com/datawrapper/api) is our new v3 REST API
+- [frontend](https://github.com/datawrapper/frontend) is our new frontend server
+- [chart-core](https://github.com/datawrapper/chart-core) is our library that renders all visualizations
+- [shared](https://github.com/datawrapper/shared) contains JavaScript utility functions used throughout datawrapper
+- [orm](https://github.com/datawrapper/orm) contains our internal database models
+
+Datawrapper was created in 2012 by [Mirko Lorenz](http://www.mirkolorenz.com/), [Nicolas Kayser-Bril](http://nkb.fr) and [Gregor Aisch](http://driven-by-data.net/). Since 2014 it is developed by the [Datawrapper GmbH](https://www.datawrapper.de/about-us/).
```

Comment:
```suggestion
Datawrapper was created in 2012 by [Mirko Lorenz](http://www.mirkolorenz.com/), [Nicolas Kayser-Bril](http://nkb.fr) and [Gregor Aisch](http://driven-by-data.net/). Since 2014 it is developed by [Datawrapper GmbH](https://www.datawrapper.de/about-us/).
```